install.sh

```diff
@@ -165,7 +165,24 @@ else
 
 fi
 
-echo -e "${COLOR_WARNING}Please restart your device to see effects...${NOCOLOR}"
-echo "Have fun, until next time, bye!"
+# Apply changes if the xochitl service is active
+if systemctl --quiet is-active xochitl.service 2> /dev/null; then
```

Comment:
This works for installing rm-hacks. However if remarkable-stylus is installed, `systemctl daemon-reload` needs to be run before restart xochitl